_data/shared_settings.yml

```diff
@@ -0,0 +1,18 @@
+# These are the values sent in the form to the contact center
+# They should *not* be translated because they are keys for the backend
+contact_page:
+  agencies:
+    cbp_jobs: Customs and Border Protection Jobs
```

Comment:
Instead of creating a separate file for this, what if we used the form value as the key in the locale settings file?

e.g.

```yaml
contact_page:
  support_request_form:
    agencies:
      'Custom and Border Protection Jobs': Emplois dans le domaine des douanes et de la protection des frontières
```

Reply:
I like it, will update